pkg/pillar/types/zedroutertypes.go

```diff
@@ -1498,7 +1523,7 @@ func IsFreeMgmtPort(globalStatus DeviceNetworkStatus, phylabelOrIfname string) b
 			!us.IsMgmt {
 			continue
 		}
-		return us.Free
+		return us.FallbackPriority == 0
```

Comment:
Might make sense to add a method "IsFree" to NetworkPortStatus that has this check inside?

Reply:
That's one of the downsides in reviewing a commit at a time I guess. That function was replaced by a differently named function which returns the actual priority. Function is used by diag and with all the changes in this PR diag now prints:
INFO: Port eth0: for EV Controller without usage-based charging
...
INFO: Port eth1: for EV Controller (priority 17)
...

(Maybe the first string should also be changed to say " (best priority 0)", and the second "(fallback priority 17")?